expression/builtin_time.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/pingcap/tidb/sessionctx/variable"
 	"github.com/pingcap/tidb/terror"
 	"github.com/pingcap/tidb/util/types"
+	"github.com/pingcap/tidb/_vendor/src/github.com/cznic/mathutil"
```

Comment:
replace it with `github.com/cznic/mathutil`

Reply:
ok